index.html

```diff
@@ -290,6 +290,39 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <div class="main">
+      <section id="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="https://formspree.io/f/xwkajypj" id="form" method="post">
+          <input
+            name="Name"
+            id="Name"
+            type="text"
+            placeholder="Full Name"
+            maxlength="20"
```

Comment:
According to the project's requirement, the name field should accept up to 30 characters but yours have a maximum length of 20. Kindly adjust to accommodate up to 30 character as specified in the requirement.